Content.Shared/CCVar/CCVars.Vote.cs

```diff
@@ -52,6 +52,18 @@ public sealed partial class CCVars
     public static readonly CVarDef<bool> VoteRestartNotAllowedWhenAdminOnline =
         CVarDef.Create("vote.restart_not_allowed_when_admin_online", true, CVar.SERVERONLY);
 
+    /// <summary>
+    ///     Whether automatic voting for a new map is enabled.
+    /// </summary>
+    public static readonly CVarDef<bool> VoteAutoMapEnabled =
+        CVarDef.Create("vote.auto_map_enabled", false, CVar.SERVERONLY);
+
+    /// <summary>
+    ///     Whether automatic voting for a new preset is enabled.
+    /// </summary>
+    public static readonly CVarDef<bool> VoteAutoPresetEnabled =
+        CVarDef.Create("vote.auto_preset_enabled", false, CVar.SERVERONLY);
```

Comment:
These CVars do not describe what they actually do. They should probably be called something like `vote.auto_map_vote_on_round_end` and have the doc comments updated to match.